content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -115,15 +115,57 @@ OpenTelemetry's [Agent Configuration](https://opentelemetry.io/docs/zero-code/ja
 
 {{% /tab %}} -->
 
-<!-- {{% tab "Go" %}}
+{{% tab "Go" %}}
 
 ## Compatibility requirements
 
+The Datadog SDK for Go supports library instrumentations written using the [Opentelemetry-Go Trace API][1]. This includes the [opentelemetry-go-contrib/instrumentation][2] libraries, however, anything that relies on metrics or logs exporters is not supported at this time.
```

Comment:
```suggestion
The Datadog SDK for Go supports library instrumentations written using the [Opentelemetry-Go Trace API][1], including the [`opentelemetry-go-contrib/instrumentation`][2] libraries, but it does not support any integration that relies on metrics or logs exporters.
```

Is integration the right word for near the end of that sentence? Or should that be "instrumentation"? I wasn't sure.